pom.xml

```diff
@@ -66,6 +66,9 @@
         <sonar-maven-plugin.version>3.9.1.2184</sonar-maven-plugin.version>
         <spotless-maven-plugin.version>2.27.2</spotless-maven-plugin.version>
         <swagger-annotations.version>2.1.0</swagger-annotations.version>
+        <jjwt-api.version>0.11.5</jjwt-api.version>
+        <jjwt-impl.version>0.11.5</jjwt-impl.version>
+        <jjwt-jackson.version>0.11.5</jjwt-jackson.version>
```

Comment:
i guess anyway we want to have same version for all 3.
So probably it would make sense to have just one maven property like 
```xml
<jjwt.version>0.11.5</jjwt.version>
```
and use it in all 3 dependencies